### DISCUSSION TITLE
azure DevOps github connection.

### DISCUSSION THREAD

**Discussion by Mytsyura** [community/community#36641] [Apps, API and Webhooks]
Im trying to install and get all permissions to this app.
<img width="509" alt="Screenshot 2022-10-19 at 14 26 39" src="https://user-images.githubusercontent.com/46034926/196678415-3c7f2f70-149d-4199-8775-ccf5bb5361db.png">
but receives next error : 
Failed to set events: commit_comment, content_reference, issues, issue_comment, pull_request, push, repository.  
does anyone know the problem?

**Comment by ossanna16** [comment]
Hey there @Mytsyura and thank you for asking a good question! I'm glad you found our community 🙂
